spec/factories/system_settings.rb

```diff
@@ -17,6 +17,7 @@
 #  confirmation_page_text_footer      :string
 #  confirmation_page_text_header      :string
 #  confirmation_page_text_link_header :string
+#  display_navbar                     :boolean          default(FALSE)
```

Comment:
Check this in or no?

Reply:
@thestephenmarshall Looks like this was inserted automatically. We didn't add the change manually so we should probably defer to @maebeale @exbinary @h-m-m  for this.

Reply:
[Annotate](https://rubygems.org/gems/annotate/versions/2.7.1) adds these schema details for both models and factories after migrations are applied. I think it's okay to check this in unless mutual-aid folks say otherwise.